build.xml

```diff
@@ -1808,11 +1816,21 @@
     <option name="ADDITIONAL_OPTIONS_STRING" value="--add-exports java.rmi/sun.rmi.registry=ALL-UNNAMED" />
   </component>
 </project>]]></echo>
-      <echo>"IDE configuration updated for use with JDK11"</echo>
+      <echo>"IDE configuration updated for use with JDK${ant.java.version}"</echo>
+      <echo>Please note that `ant realclean` does *NOT* remove IntelliJ-specific files</echo>
```

Comment:
would it be possible to have some system property which would clean IDEA-specific files too?

Does this mean that `ant realclean` in trunk would be different from `ant realclean` in older branches? Is this something we have changed here or you are just explaining how it always was?

I think if we change the behavior so it is not removed on realclean, we should backport this to older branches as well. Stuff like this should work across all branches just same.

Reply:
see comment in https://github.com/apache/cassandra/pull/2320#discussion_r1246637341